apps/prairielearn/src/tests/manualGrading.test.ts

```diff
@@ -38,92 +57,88 @@ const assessmentTitle = 'Homework for Internal, External, Manual grading methods
 const manualGradingQuestionTitle = 'Manual Grading: Fibonacci function, file upload';
 
 /**
- * @param {object} user student or instructor user to load page by
- * @returns {Promise<string>} Returns "Homework for Internal, External, Manual grading methods" page text
+ * @param user student or instructor user to load page by
+ * @returns "Homework for Internal, External, Manual grading methods" page text
  */
-const loadHomeworkPage = async (user) => {
+async function loadHomeworkPage(user: helperClient.User): Promise<string> {
   setUser(user);
   const studentCourseInstanceUrl = baseUrl + '/course_instance/1';
   const courseInstanceBody = await (await fetch(studentCourseInstanceUrl)).text();
   const $courseInstancePage = cheerio.load(courseInstanceBody);
   const hm9InternalExternalManualUrl =
     siteUrl + $courseInstancePage(`a:contains("${assessmentTitle}")`).attr('href');
-  let res = await fetch(hm9InternalExternalManualUrl);
+  const res = await fetch(hm9InternalExternalManualUrl);
   assert.equal(res.ok, true);
   return res.text();
-};
+}
 
 /**
- * @param {object} user student or instructor user to load page by
- * @returns {Promise<string>} student URL for manual grading question
+ * @param user student or instructor user to load page by
+ * @returns student URL for manual grading question
  */
-const loadHomeworkQuestionUrl = async (user) => {
+async function loadHomeworkQuestionUrl(user: helperClient.User): Promise<string> {
   const hm1Body = await loadHomeworkPage(user);
   const $hm1Body = cheerio.load(hm1Body);
   return siteUrl + $hm1Body(`a:contains("${manualGradingQuestionTitle}")`).attr('href');
-};
+}
 
 /**
  * Gets the score text for the first submission panel on the page.
- *
- * @param {cheerio.CheerioAPI} $
- * @returns {string}
  */
-const getLatestSubmissionStatus = ($) => {
+function getLatestSubmissionStatus($: cheerio.CheerioAPI): string {
   return $('[data-testid="submission-status"] .badge').first().text();
-};
-
-let iqUrl, iqId;
-let instancesAssessmentUrl;
-let manualGradingAssessmentUrl;
-let manualGradingAssessmentQuestionUrl;
-let manualGradingIQUrl;
-let manualGradingNextUngradedUrl;
-let $manualGradingPage;
-let score_percent, score_points, adjust_points;
-let feedback_note;
-let rubric_items;
-let selected_rubric_items;
-
-const submitGradeForm = async (method = 'rubric') => {
+}
+
+let iqUrl: string, iqId: string | number;
+let instancesAssessmentUrl: string;
+let manualGradingAssessmentUrl: string;
+let manualGradingAssessmentQuestionUrl: string;
+let manualGradingIQUrl: string;
+let manualGradingNextUngradedUrl: string;
+let $manualGradingPage: cheerio.CheerioAPI;
+let score_percent: number, score_points: number, adjust_points: number | null;
+let feedback_note: string;
+let rubric_items: RubricItem[];
+let selected_rubric_items: number[];
```

Comment:
Not requesting a change for this PR since I know this was re-existing code, but: putting mutable state in the top-level module scope is something we want to avoid in tests, as it makes it quite difficult to understand what's going on. Keep this in mind for future PRs.

Reply:
I'm aware of that, my plan here was just to convert the code and handle any type conflicts, not change any functionality. Given this change was small enough for git to detect a rename I'd prefer to keep that for a separate PR.

Reply:
I agree, hence my note that I'm not requesting a change here.